### PR TITLE
lint fixes; bump tchannel

### PR DESCRIPTION
**Pull request by rf**
Sorry for the 2 in 1 PR but these changes are super tiny

Fix some lint errors & bump tchannel to latest

@Raynos @jcorbin

### CODE REVIEW

**Comment by Raynos** [comment]
legit.

File: clients/index.js [hunk 1]
```diff
@@ -335,6 +335,7 @@ ApplicationClients.prototype.onRemoteConfigUpdate = function onRemoteConfigUpdat
 };
 
 ApplicationClients.prototype.updateLazyHandling = function updateLazyHandling() {
+    var self = this;
```
Comment (jcorbin) [inline clients/index.js]: wow... so this was broken before, do we not run lint out of tests like travis?

**Comment by jcorbin** [comment]
lgtm

**Comment by Raynos** [comment]
@jcorbin we do run lint; travis was not on.

**Pull request merged**